src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Configuration/OpenIdConnectConfiguration.cs

```diff
@@ -306,6 +306,7 @@ public OpenIdConnectConfiguration(string json)
         /// <summary>
         /// Gets the <see cref="ICollection{SecurityKey}"/> that the IdentityProvider indicates are to be used signing tokens.
         /// </summary>
+        [JsonIgnore]
```

Comment:
Marking a property with `[JsonIgnore]` will exclude a property from both serialization and deserialization. 

We are still using conditional serialization for the SigningKeys property: `[bool ShouldSerialize{SigningKeys}()]`.
I'm wondering what takes priority, JsonIgnore or conditional serialization? Also, should we even serialize SigningKeys to JSON i.e. should we remove `ShouldSerializeSigningKeys`?

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/d4d6d3598952410ff96ef8f38c9ca9ce74bca8bf/src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Configuration/OpenIdConnectConfiguration.cs#L526-L529

Reply:
I don't think we should serialize SigningKeys since it's not the standard.
Users can overload this method if they need to do this.

Reply:
Will remove that method.

Reply:
This method has been removed.